docs/docs/best-practices-for-orgs.md

```diff
@@ -3,13 +3,12 @@ title: Best Practices for Organizations
 issue: https://github.com/gatsbyjs/gatsby/issues/14042
```

Comment:
Te falta traducir el título del documento:

title: Best Practices for Organizations